minidump-processor/src/op_analysis.rs

```diff
@@ -48,6 +50,12 @@ pub enum OpAnalysisError {
 pub struct OpAnalysis {
     /// A string representation of the instruction for humans to read
     pub instruction_str: String,
+    /// The memory operation performed by the instruction
+    ///
+    /// Note that `None` means that memory operation could not be determined,
+    /// `Some(NoOperation)` means it was successfully determined that the crash
+    /// doesn't access memory
+    pub memory_operation: Option<MemoryOperation>,
```

Comment:
I'm not sure, but it might be nicer to drop the `Option` and just add a variant like `MemoryOperation::Unknown`?